### ISSUE TITLE
WARN Request to site https://www.example.com threw an unhandled exception. Please check the URL

### ISSUE THREAD

**Issue by welly** [harlan-zw/unlighthouse#239]
### Describe the bug

I'm testing a known working configuration against a couple of sites. Using one URL, Unlighthouse is able to resolve the URL (200 response) and complete the audit but using a second URL it fails with the following error:

```
su - node -c "unlighthouse-ci --config-file $BITBUCKET_CLONE_DIR/unlighthouse.ci.config.ts --site $UNLIGHTHOUSE_URL"
[Unlighthouse]  WARN  Request to site https://www.example.com threw an unhandled exception. Please check the URL is valid and not blocking crawlers. Request failed with status code 403
[Unlighthouse]  ERROR  Site check failed. will attempt to proceed but may fail.
[Unlighthouse] ℹ Creating Unlighthouse using config from /opt/atlassian/pipelines/agent/build/unlighthouse.ci.config.ts
[Unlighthouse] ℹ Skipping /. Invalid response from URL https://www.example.com  code: 404.
```

This also happens when using the CLI tool using the following:

 npx unlighthouse --site https://www.example.com --debug  

```
⚙ Testing Site https://www.example.com is valid.                                                                                                                                    Unlighthouse 11:11:07 AM

 WARN  Request to site https://www.example.com threw an unhandled exception. Please check the URL is valid and not blocking crawlers. Request failed with status code 403            Unlighthouse 11:11:07 AM

  at settle (.npm/_npx/944abecbf21dfffb/node_modules/axios/lib/core/settle.js:19:12)
  at BrotliDecompress.handleStreamEnd (.npm/_npx/944abecbf21dfffb/node_modules/axios/lib/adapters/http.js:599:11)
  at BrotliDecompress.emit (node:events:527:35)
  at endReadableNT (node:internal/streams/readable:1589:12)
  at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
  at Axios.request (.npm/_npx/944abecbf21dfffb/node_modules/axios/lib/core/Axios.js:45:41)
  at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  at async fetchUrlRaw (.npm/_npx/944abecbf21dfffb/node_modules/@unlighthouse/core/dist/index.mjs:2113:22)
  at async validateHost (.npm/_npx/944abecbf21dfffb/node_modules/@unlighthouse/cli/dist/shared/cli.05b1e9e4.mjs:31:65)
  at async resolved-config (.npm/_npx/944abecbf21dfffb/node_modules/@unlighthouse/cli/dist/cli.mjs:22:11)
  at async createUnlighthouse (.npm/_npx/944abecbf21dfffb/node_modules/@unlighthouse/core/dist/index.mjs:1822:3)
  at async run (.npm/_npx/944abecbf21dfffb/node_modules/@unlighthouse/cli/dist/cli.mjs:17:24)


 ERROR  Site check failed. will attempt to proceed but may fail.                                                                                                                          Unlighthouse 11:11:07 AM

```

Happy to share the site URL if this will help.

**Comment by harlan-zw** [comment]
Hi, yes this is usually due to some sort of firewall blocking access to the site OR a host using the IPV6 protocol which doesn't seem to work in some node versions.

I don't think there's any more information I can give you without the site URL, it's likely there isn't a workaround though.

**Comment by welly** [comment]
Hi @harlan-zw, thanks for your reply. The site URL is bluecross.org.uk. If you've got 5 minutes to take a look and give me any suggestions on what it might be, that would be much appreciated. I've tested this site against a couple of others on the same hosting platform - so they should possibly be the same configuration - and it seems like it's only this site which is failing. 

I checked the Cloudflare logs and couldn't even see where a request from unlighthouse was being made so it's all very odd! Any further help would be greatly appreciated! 

Thank you
